src/common/raft/integration/raft_knowhere_index.hpp

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
This file is the header actually included elsewhere in Knowhere. It exposes no RAFT symbols and does not require CUDA compilation.